spoon.rb

```diff
@@ -0,0 +1,57 @@
+STDOUT.sync = true # DO NOT REMOVE
+# Solution for https://www.codingame.com/ide/puzzle/there-is-no-spoon-episode-1
+class Spoon
+  attr_accessor :width, :height, :nodes
+
+  def initialize
+    @width = gets.to_i # the number of cells on the X axis
```

Comment:
Extract the `gets` statements outside of the method and give them as arguments to `initialize`